### PR TITLE
Updating Compat Section

### PR DESCRIPTION
**Pull request by hammy4815**
See https://github.com/gridap/GridapMakie.jl/issues/62#issuecomment-1938923425

My working dependency versions:

```
  [13f3f980] CairoMakie v0.11.8
  [5789e2e9] FileIO v1.16.2
  [1a297f60] FillArrays v1.9.3
  [e9467ef8] GLMakie v0.9.8
  [5c1252a2] GeometryBasics v0.4.10
  [56d4f2e9] Gridap v0.17.23
  [ee78f7c6] Makie v0.20.7
```

### CODE REVIEW

**Comment by hammy4815** [comment]
I think the issues above were caused from trying to compile the docs (which calls GridapMakie and then Gridap dependencies) with Julia v1.6 instead of 1.9, so I just pushed an update to try and fix the issue.

**Comment by HelgeGehring** [comment]
Yay, great to see an update :partying_face: 
Could we directly add julia 1.10 as well?

**Comment by fverdugo** [comment]
Hi @hammy4815,

the ci failed for this reason: 

"This request was automatically failed because there were no enabled runners online to process the request for more than 1 days."

This is the first time I see this problem. Any idea?

**Comment by HelgeGehring** [comment]
We probably just need to update `ubuntu-18.04` which is not supported anymore.
In #71 I made a pull request to update it to latest.

**Comment by fverdugo** [comment]
Hi @hammy4815, the PR #71 has been merged. Can you pull this changes to your fork, and then update this PR?

**Comment by hammy4815** [comment]
The only tests that are failing are:

```Julia
fig, _ , sc = plot(Γ, uh, colormap=:algae)
```

```Julia
fig, _ , plt = plot(Λ, jump(n_Λ⋅∇(uh)))
```

Because of the same error:
```Julia
can't splice MakieCore.ShadingAlgorithm into an OpenGL shader. Make sure all fields are of a concrete type and isbits(FieldType)-->true
```

Once I have this issue fixed, the tests should pass.

**Comment by hammy4815** [comment]
There seems to be some sort of bug specifically with how the shaders for GLMakie are being called, and strangely this only happens for SkeletonTriangulations and BoundaryTriangulations. The bug does not exist for CairoMakie however, so the tests work now with CairoMakie...but it's worth noting this issue for GLMakie.

**Comment by hammy4815** [comment]
@fverdugo ,

Do you have any thoughts regarding the GLMakie vs CairoMakie situation? Otherwise, could we see if it passes the tests this time?

**Comment by fverdugo** [comment]
Hi @hammy4815,

Thanks for your new attempt!

 Since you never contributed to the project I need to manually accept the execution of the tests. Could you please do some dummy change in the repo in a separate PR? This will provably speed up the debugging of this issue.

File: test/runtests.jl [hunk 1]
```diff
@@ -2,7 +2,7 @@ module TestGridapMakie
 
 using GridapMakie
 using Gridap
-using GLMakie
+using CairoMakie
```
Comment (fverdugo) [inline test/runtests.jl:5]: I see that we are not testing with GLMakie anymore.

Do the tests also pass with GLMakie witht he new compat entries?

**Comment by dnuy** [comment]
The tests also pass with Gridap v0.18.2, so I suggest to update the Gridap version in the compat section.

(I also tried using the latest Makie but that does not work.)

**Comment by JordiManyer** [comment]
This has been out-done by PR #73 . Closing this.

**Pull request closed without merging**